webapp/src/clj/lipas/backend/handler.clj

```diff
@@ -791,7 +791,7 @@
            (let [locale  (or (-> req :parameters :query :lang keyword) :en)]
```

Comment:
Legacy-api defaults to `:fi`, let's not change that. :)

http://lipas.cc.jyu.fi/api/categories